script/check_publish.sh

```diff
@@ -12,8 +12,8 @@ source "$SCRIPT_DIR/common.sh"
 
 function check_publish() {
   local failures=()
-  for package_name in "$@"; do
-    local dir="$REPO_DIR/packages/$package_name"
+  for dir in $(pub global run flutter_plugin_tools list --plugins="$1"); do
+    local package_name=$(basename "$dir")
```

Comment:
This tooling changes are similar to the ones made in flutter/plugins, right?

Reply:
Yes.